src/docs/patterns/tables.mdx

```diff
@@ -0,0 +1,469 @@
+    ]}
+    captionsUnderImages={['', 'DONT:']}
+/>
+
+## Property Name
+
+In tables, each data point is labeled only once in the table header. In lists, however, the values are either clear enough on their own without explanations, or they have to be included in each list item.
+
+<ImageGrid
+    images={[ImplicitValueTable]}
+    captionsUnderImages={[`The values are explained by the table header row.`]}
+    gridContainerProps={{ style: { justifyContent: 'flex-start' } }}
+/>
+<ImageGrid
+    images={[ImplicitValueList1, ImplicitValueList2]}
+    captionsUnderImages={[
+        `DONT: It is not clear what those values in the subtitle mean.`,
+        `CAUTION: Be careful when calling out property names in lists, as they weaken the scanability.`,
+    ]}
+/>
+
+## Consumer-Facing Applications
+
+Avoid using tables in consumer-facing applications. Lists mentally match real-world individual objects, while tables can be more abstract. Use tables only if your users really need to examine technical details.
+
+<Divider />
+
+# Responsive
+
+In general, **we do not encourage tables on mobile platforms**, as table-related interactions tend to be complicated for small viewports, and mobile phones also often lack the computational power to process massive data sets.
+
+<ImageGrid
+    images={[TableOnMobile]}
+    regularWidth
+    caption={`Users can only see a very small portion of the table through mobile devices.`}
+/>
+
+## Understanding Users' Goal
+
+<ImageGrid images={[UserInField]} regularWidth caption={`While on foot, it is very hard to examine small details.`} />
+
+Before you design a table structure for a mobile application, you should conduct user research and have a clear understanding of users' intentions. When they land on this page, do they need to do complicated cross-comparisons between multiple rows and columns? Is a full table necessary?
+
+For an industrial mobile application, users would often be on foot and out in the field when they interact with the app. They are paying more attention to their surroundings, so keep the mobile app interaction minimal.
+
+## Breaking Down Tables into Lists
+
+If a user only needs a few parameters to identify the correct data entry and make decisions, a list with a few selected data points would suffice.
+
+<ImageGrid
+    images={[HomePagePreview, TableStackedAsList]}
+    captionsUnderImages={[
+        `On a crowded page, show a preview of the table content, with the option to view the full table in a dedicated page.`,
+        `Stack each property into a list item, with the ability to click into a list item to view the full item details.`,
+    ]}
+/>
+
+## Pagination
+
+If tables become necessary in your mobile application, pagination is especially helpful to reduce the load of both the server and the client-side render.
```

Comment:
I think there is probably a stronger case for infinite scroll with virtualization (or progressive loading) on mobile versus pagination. Vertical scrolling is a very intuitive interaction on mobile. I don't think I would like having to hit the next button 9 times to get to the end of my list...I would rather fling the heck out of the screen and zoom to the bottom.

Reply:
I will add this back... I didn't add inf scroll bc Ian hated it.